### PR TITLE
fix: disable selinux label when mounting the volume for the playground

### LINKED ISSUE #456: Playground cannot start on Linux and MacOS applehv

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #456

### What does this PR do?

Change options while starting the playground server

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#456

### How to test this PR?

Start the playground for a model on Linux or MacOS applehv

### CODE REVIEW

File: packages/backend/src/managers/playground.ts [hunk 1]
```diff
@@ -212,6 +212,7 @@ export class PlayGroundManager {
             Type: 'bind',
           },
         ],
+        SecurityOpt: [DISABLE_SELINUX_LABEL_SECURITY_OPTION],
```
Comment (axel7083) [inline packages/backend/src/managers/playground.ts:215]: What are the implication of this security option ? Why does it is required ? I cannot really understand the link with applehv from the documentation https://docs.podman.io/en/v4.6.1/markdown/options/security-opt.html

Comment (jeffmaury) [reply packages/backend/src/managers/playground.ts:215]: The volumes are not readable if this option is not set on Linux and applehv. Inspired from https://github.com/containers/podman/issues/3683

Comment (axel7083) [reply packages/backend/src/managers/playground.ts:215]: So we might want to include this option on windows or we don't care ?

Comment (jeffmaury) [reply packages/backend/src/managers/playground.ts:215]: It is set for all platforms, I prefer to have a single general flow

**Pull request merged**